modules/ossm-cr-istio-global.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-cr-istio-global_{context}"]
+= Istio global example
+
+Here is an example that illustrates the Istio global parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
+
+[NOTE]
+====
+In order for the 3scale Istio Adapter to work, `disablePolicyChecks` must be `false`.
+====
+
+[source,yaml]
+----
+  istio:
+    global:
+      hub: `maistra/` or `registry.redhat.io/openshift-istio-tech-preview/`
+      tag: 0.11.0
+      proxy:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 500m
+            memory: 128Mi
+      mtls:
+        enabled: false
+      disablePolicyChecks: true
+      policyCheckFailOpen: false
+      imagePullSecrets:
+        - MyPullSecret
+----
+
+[NOTE]
+====
+See the OpenShift documentation on xref:../../scalability_and_performance/recommended-host-practices.html#recommended-node-host-practices_[Scalability and performance] for additional details on CPU and memory resources for the containers in your pod.
+====
+
+.General parameters
+|===
+|Parameter |Description |Values |Default value
+
```

Comment:
Not sure if it warrants being documented here, but there's a field not mentioned here, dnsRefreshRate. This field is the number of seconds to wait between domain name service refresh requests from Envoy. Previously this was set at 5 seconds. It has since been changed to 300 seconds in order to avoid problems with DNS flooding.

Reply:
This change *should* be invisible to users. It was fixed in MAISTRA-573

Reply:
@brian-avery Is changing the refresh rate (now that the DNS flooding issue has been taken care of) something that users do frequently?

Reply:
Haven't heard of anyone doing it, but I haven't talked to a lot of users. My guess is no.

Reply:
I'm OK with leaving it out if we don't want to encourage people to play with it.